src/Init/Data/Order/Lemmas.lean

```diff
@@ -22,10 +22,41 @@ section AxiomaticInstances
 public instance (r : α → α → Prop) [Asymm r] : Irrefl r where
   irrefl a h := Asymm.asymm a a h h
 
-public instance {r : α → α → Prop} [Total r] : Refl r where
+public instance (r : α → α → Prop) [Total r] : Refl r where
   refl a := by simpa using Total.total a a
 
-public instance Total.asymm_of_total_not {r : α → α → Prop} [i : Total (¬ r · ·)] : Asymm r where
```

Comment:
There are backward compatibility reasons why I'd prefer to keep this instance: Many lemmas were previously stated in terms of `Total (¬ r · ·)` and are now stated in terms of `Asymm`. This instance makes sure that things don't break.